### PR TITLE
Fix lisk-client export - Closes #7338

### LINKED ISSUE #7338: Lisk desktop tests fail when lisk-client is updated to v6
### Expected behavior

Configure Lisk desktop tests environment to handle `buffer` for both browser and nodejs

### Actual behavior

- When we update `lisk-client` dependency to its latest version the tests starts to fail
- The root cause of failing tests are due to the changes in the way lisk-client is being shipped from version >= v6
  - The global buffer is an instance of browser buffer https://github.com/LiskHQ/lisk-sdk/commit/f0c25ef51a69f232dcd95782306e78f2edf7316e
  - Since the jest setup is configured to work with nodejs buffer, if we update lisk-client to its latest version the lisk-client functions starts to fail
  
### Steps to reproduce

- Checkout development branch
- Update `lisk-client` to latest version `v6`
- Run `npm install`
- Run `npm run test`

### Which version(s) does this affect? (Environment, OS, etc...)
Lisk Desktop v3

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #7338

### How was it solved?

- main is used mainly for library, so updating the default to node.js version
- Add `browser` field to specify the usage during the browser build
- Update plugin bundles to `parcels` to read the browser field and make it easy to update the version

### How was it tested?
- Run `npm run start:web` on faucet plugin and ensure UI build is working
- Check `lisk-client/index.html` and see `lisk` is usable in the browser console

### CODE REVIEW

**Review by ishantiw — APPROVED** [review]
- Ran faucet plugin and it builds and loads without any error
- Tested `lisk-client/index.html` where `lisk` is accessible with all the expected objects

File: framework-plugins/lisk-framework-dashboard-plugin/package.json [hunk 1]
```diff
@@ -34,11 +34,12 @@
 		"prebuild": "rm -r dist-node/* || mkdir dist-node || true",
 		"build": "npm run build:node && npm run build:web",
 		"build:node": "tsc",
-		"build:web": "node scripts/build.js --jsx react",
-		"start:web": "NODE_ENV=development node scripts/start.js",
+		"build:web": "parcel build --dist-dir build public/index.html",
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-dashboard-plugin/package.json:37]: why usage of `build` command twice 🤔

File: framework-plugins/lisk-framework-faucet-plugin/package.json [hunk 2]
```diff
@@ -35,11 +35,12 @@
 		"prebuild": "rm -r dist-node/* || mkdir dist-node || true",
 		"build": "npm run build:node && npm run build:web",
 		"build:node": "tsc",
-		"build:web": "node scripts/build.js --jsx react",
-		"start:web": "NODE_ENV=development node scripts/start.js",
+		"build:web": "parcel build --dist-dir build public/index.html",
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-faucet-plugin/package.json:38]: same here

**Review by ManuGowda — APPROVED** [review]

File: framework-plugins/lisk-framework-dashboard-plugin/package.json (hunk 1 above)
Comment (shuse2) [reply framework-plugins/lisk-framework-dashboard-plugin/package.json:37]: second one is specifying the `dist-dir` first one is part of the command `parcel build`

**Pull request merged**